transport/src/main/java/io/netty/bootstrap/Bootstrap.java

```diff
@@ -216,6 +217,17 @@ public Bootstrap clone() {
         return new Bootstrap(this);
     }
 
+    /**
+     * Returns a deep clone of this bootstrap which has the identical configuration except that it uses
+     * the given {@link EventLoopGroup}. This method is useful when making multiple {@link Channel}s with similar
+     * settings.
+     */
+    public Bootstrap clone(EventLoopGroup group) {
```

Comment:
@normanmaurer - At this point I'm guessing you have thought about it and want to leave it as is?